lampo-grpc/src/lib.rs

```diff
@@ -0,0 +1,30 @@
+use lampod::LampoDaemon;
+use std::sync::Arc;
+use tonic::transport::Server;
+
+pub const FILE_DESCRIPTOR_SET: &[u8] = tonic::include_file_descriptor_set!("descriptor");
+
+mod server;
+
+use server::lnrpc::lightning_server::LightningServer;
+use server::GrpcServer;
+
+pub async fn run_grpc(
```

Comment:
are you sure that this is LND compatible? looks like that the AUTH path it is missing?

I would like to have an version that can work with the library https://crates.io/crates/lnd_grpc_rust without they the user know that it is not LND